tests/components/hisense_aehw4a1/test_init.py

```diff
@@ -0,0 +1,94 @@
+"""Tests for the Hisense AEH-W4A1 init file."""
+from unittest.mock import patch
+
+from pyaehw4a1 import exceptions
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.components import hisense_aehw4a1
+from homeassistant.setup import async_setup_component
+
+from tests.common import mock_coro
+
+
+async def return_async_value(val):
+    """Return given value for a mocked async function."""
+    return val
+
+
+async def test_creating_entry_sets_up_climate_discovery(hass):
+    """Test setting up Hisense AEH-W4A1 loads the climate component."""
+    with patch(
+        "homeassistant.components.hisense_aehw4a1.config_flow.AehW4a1.discovery",
+        return_value=return_async_value(["1.2.3.4"]),
```

Comment:
Sorry, I didn't know that the library uses asyncio. I thought it was doing sync I/O. Then it's ok to use mock_coro. Or keep it like it is now.

Reply:
I'll switch to moko to ensure uniformity with HA.
I started writing in python about a month ago, but I learned the most important things in the last five days. Thanks!